app/src/main/java/io/github/gsantner/memetastic/activity/MainActivity.java

```diff
@@ -373,6 +376,21 @@ public boolean handleBarClick(MenuItem item) {
                 _toolbar.setTitle(R.string.memelist_data_mode__saved);
                 break;
             }
+
+            case R.id.action_mode_hidden:{
```

Comment:
can you set visibility hidden if the list of hidden is empty?

Reply:
Okay, if there's no meme hidden yet, the nav bar should not show the hidden tab right?

Reply:
y